composer.json

```diff
@@ -6,17 +6,18 @@
     "minimum-stability": "dev",
     "require": {
         "php": ">=7.2.5",
-        "symfony/config": "^4.4 | ^5.0 | ^6.0",
-        "symfony/dependency-injection": "^4.4 | ^5.0 | ^6.0",
-        "symfony/http-kernel": "^4.4 | ^5.0 | ^6.0",
-        "symfony/routing": "^4.4 | ^5.0 | ^6.0",
+        "ext-json": "*",
+        "symfony/config": "^5.4 | ^6.0",
+        "symfony/dependency-injection": "^5.4 | ^6.0",
+        "symfony/http-kernel": "^5.4 | ^6.0",
+        "symfony/routing": "^5.4 | ^6.0",
         "symfony/deprecation-contracts": "^2.2 | ^3.0"
     },
     "require-dev": {
         "doctrine/orm": "^2.7",
         "doctrine/persistence": "^2.0",
-        "symfony/framework-bundle": "^4.4 | ^5.0 | ^6.0",
-        "symfony/phpunit-bridge": "^5.0 | ^6.0",
+        "symfony/framework-bundle": "^5.4 | ^6.0",
+        "symfony/phpunit-bridge": "^5.4 | ^6.0",
         "vimeo/psalm": "^4.3"
     },
     "conflict": {
```

Comment:
We should be able to remove this conflict now if i'm not mistaken

Reply:
Hm, according to the version constraint we have so far: `^4.4 | ^5.0 | ^6.0"` - it sounds like we could drop this even before this PR as `<4.4` is not an option anymore, unless I'm missing something